packages/documentation/src/stories/components/accordion/accordion.stories.ts

```diff
@@ -99,39 +100,38 @@ export const DefaultCollapsedPanels: Story = {
   `,
 };
 
-// Used only in testing for now.
 export const Nested: Story = {
   render: () => html`
-    <post-accordion>
+    <post-accordion heading-level="4">
       <post-accordion-item>
-        <span slot="header">Titulum 1</span>
+        <span slot="header">Titulum 1 <code>h4</code></span>
```

Comment:
If you use the storybook logic to override args on the story level, you should dynamically output the heading-level here as well.